src/pspm_split_sessions.m

```diff
@@ -1,17 +1,17 @@
-function varargout = pspm_split_sessions(datafile, markerchannel, options)
+function varargout = pspm_split_sessions(datafile, options)
 % ● Description
 %   pspm_split_sessions splits experimental sessions/blocks, based on
 %   regularly incoming markers, for example volume or slice markers from an
 %   MRI scanner, or based on a vector of split points that is defined in
 %   terms of markers. The first and the last marker will define the start of
 %   the first session and the end of the last session.
 % ● Format
-%   newdatafile = pspm_split_sessions(datafile, markerchannel, options)
+%   newdatafile = pspm_split_sessions(datafile, options)
 % ● Arguments
 %            datafile:  a file name
-%       markerchannel:  (optional)
-%                       number of the channel containing the relevant markers.
 %   ┌─────────options:
+%   ├.marker_chan_num: [integer] number of the channel holding the markers.
```

Comment:
Would it be useful to keep the checking for `options.marker_chan_num` in the code? I saw the checking for `markerchannel` is removed, but I did not find corresponding checking for `options.marker_chan_num`.